app/src/main/java/com/example/drawingApp/DialogObject.kt

```diff
@@ -0,0 +1,127 @@
+package com.example.drawingApp
+
+import android.content.Context
+import android.view.LayoutInflater
+import android.view.View
+import android.widget.EditText
+import android.widget.TextView
+import android.widget.Toast
+import androidx.appcompat.app.AlertDialog
+import androidx.core.view.isVisible
+import androidx.core.widget.addTextChangedListener
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.RecyclerView
+import com.google.android.material.bottomsheet.BottomSheetDialog
+
+object DialogObject {
```

Comment:
Is this a utility class? I think so. `DialogUtility`?

Reply:
renamed